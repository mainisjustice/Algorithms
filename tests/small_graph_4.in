7
0 5 4 12 10 4 12
5 0 0 6 5 0 1
4 0 0 9 13 11 14
12 6 9 0 0 13 2
10 5 13 0 0 3 11
4 0 11 13 3 0 0
12 1 14 2 11 0 0