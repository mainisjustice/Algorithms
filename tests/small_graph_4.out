0 5 4 8 7 4 6 
5 0 9 3 5 8 1 
4 9 0 9 11 8 10 
8 3 9 0 8 11 2 
7 5 11 8 0 3 6 
4 8 8 11 3 0 9 
6 1 10 2 6 9 0 